docs/user-manual/en/amqp-server-connections.md

```diff
@@ -0,0 +1,230 @@
+# Broker Connections
+[//]: <> (This line is commnented out from the output: In case we add other protocol types on server connections, this chapter will need to renamed and some context changed )
+
+Server connections were introduced in ActiveMQ Artemis at version 2.16.0. At the time of the implementation we introduced the capability of Broker Connections.
+
+Instead of waiting for clients to connect, the ActiveMQ Artemis Broker will be proactive and create a connection to another endpoint on a specific protocol without.
+
+At the moment this feature the only supporting protocol is AMQP however it could be eventually expanded to other protocols.
+
+Server connections are configured through `broker.xml`
+
+```xml
+<broker-connections>
+    ...
+</broker-connections>
+```
+
+# AMQP Server Connections
+ActiveMQ Artemis has the capability of initiating a connection towards other endpoints using the AMQP Protocol.
+
+That means the broker will connect to another AMQP server (not necessarily ActiveMQ Artemis) and create elements towards that connection.
+
+You can define an amqp broker connection with the XML element amqp-connection, with the following arguments
+
+- uri : tcp://host:myport (this is a required argument)
+- name : The name of the connection used for management purposes
+- user : you can optionally specify the user to connect on the endpoint. 
+- password: you can optionally specify the password to connect on the endpoint
+- retry-interval: how long to wait in milliseconds before retrying a connection after an error. (default: 5000)
+- reconnect-attempts: default is -1 meaning infinite
+
+This is an example of an amqp broker connection definition:
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker" retry-interval="100" reconnect-attempts="-1" user="john" password="doe">
+        <mirror  queue-removal="true" target-mirror-address="$mirror" queue-creation="false" message-acknowledgements="true" source-mirror-address=""/>
+        <peer match="queues.#"/>
+    </amqp-connection>
+</broker-connections>
+```
+*Important:* The target endpoint needs permission on all operations you configure. So if you're using a security manager make sure you use an user with enough privileges to perform your configured operations.
+
+# AMQP Server Connection elements
+We currently support the following types of elements on a AMQP Server Connection:
+
+- Senders
+    - Messages received on specific queues will be transferred to another endpoint
+- Receivers
+    - The broker will pull messages from another endpoint
+- Peers
+    - The broker will create both senders and receivers to another endpoint that knows how to handle it. This is currently implement by qpid-dispatch.
```

Comment:
Perhaps "The broker will create a sender to and receiver from the peer on the matching address, establishing the broker queue as a waypoint for the address. This is currently supported by Qpid Dispatch router."